src/api/layers/sender.layer.ts

```diff
@@ -234,13 +234,18 @@ export class SenderLayer extends ListenerLayer {
   /**
    * Sends image message
    * @category Chat
-   * @param to Chat id
-   * @param base64 File path, http link or base64Encoded
+   * @param to ID of the chat to send the image to
+   * @param base64 File path, HTTP link, or a base64-encoded data URI (with mime type)
```

Comment:
not sure if it really supports all these formats, I think it is only base64?
```suggestion
   * @param base64 A base64-encoded data URI (with mime type)
```